### PR TITLE
Fix/x stream 555 conditionnaly show info button

### PR DESCRIPTION
**Pull request by P0ppoff**
## Checklist

Definition Of Review

-   [x] Own code review done
-   [x] Write message in teams channel

Definition Of Mergeable

-   [x] 2 approves
-   [x] 1 functional review - US has been tested
-   [x] Every comment is handled - blocking ones have been resolved by reviewer
-   [x] Design OK
-   [x] Can be tested by POs
-   [x] PR was introduced during daily meeting

### CODE REVIEW

File: libs/ui/src/components/RecordEdition/EditRecordContent/uiElements/StandardField/StandardFieldValue/DSRangePickerWrapper.test.tsx [hunk 1]
```diff
@@ -286,16 +286,16 @@ describe('DSRangePickerWrapper', () => {
...
+            const clearButton = screen.getByRole('button');
+            await user.click(clearButton);
 
             expect(mockOnChange).toHaveBeenCalledTimes(1);
             expect(mockOnChange).toHaveBeenCalledWith([expect.any(Object), expect.any(Object)], raw_value);
             expect(mockHandleSubmit).toHaveBeenCalledTimes(1);
             expect(mockHandleSubmit).toHaveBeenCalledWith('', state.attribute.id);
         });
 
-        test('Should hide clear icon when value is inherited , but not override', async () => {
```
Comment (Delmotte-Vincent) [inline libs/ui/src/components/RecordEdition/EditRecordContent/uiElements/StandardField/StandardFieldValue/DSRangePickerWrapper.test.tsx:298]: **praise:** oeil de linx

**Review by Delmotte-Vincent — APPROVED** [review]

**Review by TdyP — APPROVED** [review]

File: libs/ui/jest.config.js [hunk 2]
```diff
@@ -3,7 +3,6 @@
 // License text available at https://www.gnu.org/licenses/lgpl-3.0.txt
 module.exports = {
     testEnvironment: 'jest-environment-jsdom',
-    testTimeout: 15000,
```
[the remark this answers is not in the document]
Comment (P0ppoff) [reply libs/ui/jest.config.js:6]: Il y en a un deuxième à la ligne 19

[the remark this answers is not in the document]
Comment (P0ppoff) [reply libs/ui/jest.config.js:6]: j'ai fait dans un commit distint

**Review by evoiron — APPROVED** [review]

File: libs/ui/jest.config.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (TdyP) [reply libs/ui/jest.config.js:6]: j'avais pas fait gaffe au nom du commit, autant pour moi!

**Pull request merged**